tests/helpers/toKebabCase.js

```diff
@@ -0,0 +1,3 @@
+export default function(string) {
```

Comment:
Could be replace with `kebabCase` from [lodash](https://lodash.com/docs/4.17.15#kebabCase)

Reply:
Needs a decision first if we want to go with lodash / lodash-es / whatever. Keep it like it is for now.